crates/typst/src/foundations/calc.rs

```diff
@@ -908,6 +910,87 @@ pub fn quo(
     Ok(floor(dividend.apply2(divisor.v, Div::div, Div::div)))
 }
 
+/// Calculates the p-norm of a sequence of values.
+///
+/// ```example
+/// #calc.norm(1, 2, -3, 0.5)
+/// #calc.norm(p: 3, 1in, 2cm)
+/// #calc.norm(3em, 4em)
+/// ```
+#[func(title = "P-Norm")]
```

Comment:
```suggestion
#[func(title = "𝑝-Norm")]
```
I think U+1D45D 𝑝 would be more appropriate here? At least I think it should be lowercase.